bentoml/pytorch.py

```diff
@@ -35,7 +30,16 @@
         """  # noqa
     )
 
-infer_mode_compat = torch.__version__.startswith("1.9")
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    import importlib_metadata
+
+_ModelType = t.Union["torch.nn.Module", "torch.jit.ScriptModule"]  # type: ignore[reportPrivateUsage]
+
+_torch_version = importlib_metadata.version("torch")
+
+infer_mode_compat = _torch_version.startswith("1.9")
 
```

Comment:
why we want to decide pytorch version this way?

Reply:
pyright fix

Reply:
We should just have a utility function that gets a module version.